docs/Chromaticqa-Guide.md

```diff
@@ -0,0 +1,20 @@
+# ChromaticQA
```

Comment:
This documentation is good, but Chromatic is actually more useful in Psammead. I would move this there and reference it in existing documentation pages.